islandora_solr_metadata.routing.yml

```diff
@@ -0,0 +1,47 @@
+islandora_solr_metadata.metadata_display:
+  path: 'admin/config/islandora/search/islandora_solr/metadata'
+  defaults:
+    _title: 'Metadata Display'
+    _controller: '\Drupal\islandora_solr_metadata\Controller\DefaultController::islandoraSolrMetadataAdminPageCallback'
+  requirements:
+    _permission: 'administer islandora_solr_metadata'
+  options:
+    _admin_route: TRUE
+islandora_solr_metadata.config_1:
```

Comment:
Should use more descriptive names than config_1 and config_2: perhaps include what is being configured.